hack/common.sh

```diff
@@ -336,10 +344,13 @@ os::build::internal::build_binaries() {
 }
 readonly -f os::build::build_binaries
 
-# Generates the set of target packages, binaries, and platforms to build for.
-# Accepts binaries via $@, and platforms via OS_BUILD_PLATFORMS, or defaults to
-# the current platform.
-function os::build::export_targets() {
+# Generates the set of target packages and binaries (as full go package) to build
+# a for given platform.  First argument is platform, remaining arguments are
+# targets.  Targets can be given as full Go package path or as basenames.
+function os::build::export_targets_and_binaries() {
+  local platform=${1}
```

Comment:
We prefer not to brace positional args smaller than `10`. We do prefer to quote, though.